packages/plugins/typescript/operations/tests/ts-documents.spec.ts

```diff
@@ -2644,6 +2644,58 @@ describe('TypeScript Operations Plugin', () => {
       expect(o).toContain(`__typename?: 'Information_Entry', id: Information_EntryType,`);
     });
 
+    it('Should produce valid output with preResolveTypes=true and enums with suffixes set', async () => {
```

Comment:
this test we have suffix disabled 🤔 
```suggestion
    it('Should produce valid output with preResolveTypes=true and enums with no suffixes', async () => {
```